app/src/main/kotlin/com/wire/android/ui/authentication/create/username/CreateAccountUsernameViewModel.kt

```diff
@@ -29,10 +32,12 @@ class CreateAccountUsernameViewModel @Inject constructor(
         private set
 
     fun onUsernameChange(newText: TextFieldValue) {
+        val newValidTextString = newText.text.replace(Regex(USERNAME_FORBIDDEN_CHARACTERS_REGEX), "").take(USERNAME_MAX_LENGTH)
```

Comment:
Can this be replaced with `ValidateUserHandleUseCase` ?
or here the not allowed characters are simply ignored?